src/container/login/Login.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import Input from 'components/shared/input/Input';
+import Button from 'components/shared/button/Button';
+import ExperiousLogo from 'assets/logo/logo1.png';
+import VirtualEscapeLogo from 'assets/logo/logo2.png';
+import { useAuthContext } from 'context/auth-context/AuthContainer';
+import { ReactComponent as ThumbIcon } from 'assets/thumb.svg';
+import { MAIN_ROUTE } from 'container/routes/Constants';
+import styles from './Login.module.scss';
+
+const LogIn = () => {
+  const { setLogin } = useAuthContext();
+
+  const navigate = useNavigate();
+  const [isLenMaxed, setIsLenMaxed] = useState(false);
+  const onChangeHandler = (e) => {
+    if (e.target.value.length === 8) {
+      setIsLenMaxed(true);
+    } else {
+      setIsLenMaxed(false);
+    }
+  };
+
+  const onClickHandler = () => {
+    if (isLenMaxed) {
+      setLogin();
+      navigate(MAIN_ROUTE);
+    }
+  };
+  return (
+    <div className={styles['main-login-container']}>
+      <div className={styles['logo-container']}>
```

Comment:
this is part of Navigation? we should have props for navigation where we can mention what to render.
If we want to render only logo then shouldLogoRender = true